tests/integration-tests/tests/scaling/test_mpi.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+import pytest
+
+from remote_command_executor import RemoteCommandExecutor
+from tests.common.mpi_common import _test_mpi
+from tests.common.schedulers_common import get_scheduler_commands
+
+INSTANCES_TO_SLOTS_MAP = {"c5n.18xlarge": 72, "c5.xlarge": 4}
+
+
+@pytest.mark.regions(["us-west-2"])
+@pytest.mark.instances(["c5.xlarge", "c5n.18xlarge"])
+@pytest.mark.schedulers(["slurm", "sge"])
+@pytest.mark.oss(["alinux", "centos7", "centos6", "ubuntu1604"])
+@pytest.mark.nodewatcher
+def test_mpi(scheduler, region, os, instance, pcluster_config_reader, clusters_factory):
+    scaledown_idletime = 3
+    max_queue_size = 5
+    slots_per_instance = INSTANCES_TO_SLOTS_MAP[instance]
```

Comment:
I don't know how many instance types we want test, but you could pull this file from S3 instead of hard coding it.

Reply:
good idea, done!